Userland/Libraries/LibWeb/MimeSniff/MimeType.h

```diff
@@ -14,6 +14,27 @@ namespace Web::MimeSniff {
 
 bool is_javascript_mime_type_essence_match(StringView);
 
+// https://mimesniff.spec.whatwg.org/#javascript-mime-type
+// A JavaScript MIME type is any MIME type whose essence is one of the following:
+static constexpr Array s_javascript_mime_type_essence_strings = {
```

Comment:
Why did this move to the .h file? It's only used in a single .cpp file as far as I can tell.

(`static` is for things in .cpp files, `inline` is for things in .h files. But this one can stay `static`, it should just move to the cpp file.)